doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+            Configure an editor for Subversion
+            <br/>
+            Subversion will open an editor when we try to create a branch or commit changes
+            to the repository as it generally requires a commit log message to accompany 
+            the change.
+            <br/>
+            The default editor is vi (or gedit when running the Fcm GUI, the Fcm GUI will 
+            not be covered in this tutorial).
+            <br/>
+            To change the default editor set the environment variable SVN_EDITOR
+            <br/>
+            Note that the editor must be able to run in the foreground.
+            <br/>
+            For example, we can add one of the followings in our ~/.kshrc (ksh) or 
+            ~/.bashrc (bash):
+            <br/>
+            # GVim
+            <br/>
+            <code>export SVN_EDITOR='gvim -f'</code>
+            <br/>
+            # Emacs
+            <br/>
+            <code>export SVN_EDITOR=emacs</code>
+            <br/>
+            # gedit
+            <br/>
+            <code>export SVN_EDITOR=gedit</code>
+            <br/>
+            # Kate
+            <br/>
+            <code>export SVN_EDITOR=kate</code>
+            <br/>
+            The editor-cmd option is in the [helpers] section of the user's 
+            ~/.subversion/config file.
+            <br/>
+            For further information on this please see the file.
+            <br/>
+        </ul>
+        </p>
+        <h1><a id="S5">Setting-up your own repository</a></h1>
+        <ul>
+            <div class="well">
+                <h5>General</h5>
+                Always consider how large it is thought that the repository will grow to.
+                <br/>
+                With this in mind carefully where the repository is to be located. For Example:
+                <br/>
+                <ul>
+                    <li>Only locally as it is only ever for personal use and it is small.</li>
+                    <li>On a shared storage area for the group to all working on.</li>
+                    <li>As part of a Fcm project area as all need to be able to access it, note the 
+                        repository can have limited commit access which is done at set-up and can be 
+                        altered as appropriate via the Trac pages (see the Specific 
+                        <a href="fcm_specifics#S3">Trac</a> tutorial for more on Trac).
+                    </li>
+                </ul>
+                <br/>
+                The Fcm repository is backed up so a personal back-up is not required.
+                <br/>
+                Any local repository that has been personally created will not necessarily be 
+                backed up so the responsiblility is on the person.
+                <br/>
+                <br/>
+                <h5>Creating your own central Fcm repository</h5>
+                Normally the Fcm team will help with setting up the initial repository.
```

Comment:
This is quite specific to us. Perhaps something like Normally, you would contact your local FCM admin team for help with setting up the initial central repository